.travis.yml

```diff
@@ -7,9 +12,12 @@ cache:
 install:
   - pip install pipenv==8.3.2
   - pipenv install --dev --deploy
+  - npm install -g phantomjs-prebuilt
+  - docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
 
 script:
   - pipenv check --style . --max-line-length 120
+  - make start_services setup acceptance_tests stop_services
```

Comment:
Missing the docker network on Travis: `docker network create rasrmdockerdev_default`